src/app/api/api_v1/endpoints/detections.py

```diff
@@ -112,10 +116,36 @@ async def fetch_detections(
     if UserRole.ADMIN in token_payload.scopes:
         return [elt for elt in await detections.fetch_all()]
 
-    cameras_list = await cameras.fetch_all(("organization_id", token_payload.organization_id))
+    cameras_list = await cameras.fetch_all(filter_pair=("organization_id", token_payload.organization_id))
     camera_ids = [camera.id for camera in cameras_list]
 
-    return await detections.get_in(camera_ids, "camera_id")
+    return await detections.fetch_all(in_pair=("camera_id", camera_ids))
+
+
+@router.get("/unacknowledged/from", status_code=status.HTTP_200_OK, summary="Fetch all the unacknowledged detections")
+async def fetch_unacknowledged_detections(
+    from_date: Optional[datetime] = None,
+    detections: DetectionCRUD = Depends(get_detection_crud),
+    cameras: CameraCRUD = Depends(get_camera_crud),
+    token_payload: TokenPayload = Security(get_jwt, scopes=[UserRole.ADMIN, UserRole.AGENT, UserRole.USER]),
+) -> List[Detection]:
+    telemetry_client.capture(token_payload.sub, event="unacknowledged-fetch")
+
+    all_unck_detections = await detections.fetch_all(filter_pair=("is_wildfire", None))
+    if from_date is not None:
+        all_unck_detections = [detection for detection in all_unck_detections if detection.created_at >= from_date]
```

Comment:
Better to extend again the fetch_all to have this filter in SQL directly

Reply:
I will add it in my last PR (there is refactor of this endpoint there)

Reply:
Alright, but please next time, 1 PR = 1 focus. If we put fixes of PR in other PRs, it's tricky to review & we lose what we've tried to have so far: every single commit on "main" works (that's why we squash commit of a PR)